Quadriga/src/main/java/edu/asu/spring/quadriga/dao/workspace/impl/WorkspaceDAO.java

```diff
@@ -226,9 +229,8 @@ public boolean deleteWorkspace(String wsId) {
     }
 
     /*
-     * ================================================================= 
-     * Private Methods 
-     * =================================================================
+     * ================================================================= Private
+     * Methods =================================================================
      */
 
     /**
```

Comment:
did you change anything here? Unless a file is totally off with the formatting, please try not to commit files that were just reformatted, or make a note if you actually changed anything in them.